crimsobot/bot.py

```diff
@@ -190,21 +191,24 @@ async def on_message(self, message: discord.Message) -> None:
         if self.is_banned(message.author) or message.author.bot:
             return
 
-        # send DMs to the bot that are not bot commands to the specified channel
-        is_dm = isinstance(message.channel, discord.DMChannel)
+        # send DMs to the bot (that are not bot commands) to the specified channel
+        is_dm = message.channel.type == discord.ChannelType.private
         if is_dm and not message.content.startswith(('>', '.')):
+            # grab any attachments in message to send as links
+            links = []
             try:
-                link = message.attachments[0].url
+                for attachment in message.attachments:
+                    links.append(attachment.url)
             except IndexError:
-                link = ''
+                pass
```

Comment:
don't need the try/except anymore, can just be the for loop.

Reply:
Fixed in latest push